docs/How  to use the epos plugin.md

```diff
@@ -56,10 +56,25 @@ To make sure its the one you are looking for you can make a curl to the API with
 Now take a look at terminal 1, it will display a table with important findability-related terms, one of them is the title, so you can make sure the item is the one that you want,
 (If the table displays a lot of ... items try to make the window wider and retry the test)
 
+#### 2. Use the script searcher.py or the -q parameter of the fair-eva.py script
```

Comment:
As I noted below, I don't see the need of maintaining the `searcher.py`, it is integrated in the `fair-eva.py`